ocp_install_env.sh

```diff
@@ -74,7 +74,7 @@ controlPlane:
     baremetal: {}
 platform:
   baremetal:
-    api_vip: ${API_VIP}
+    apiVIP: ${API_VIP}
```

Comment:
@cybertron Should we pull this out entirely? Are there cases where the installer's logic to lookup in dns won't work and we should look at the env variable?

Reply:
It should be safe to just remove this. Right now both the virt and baremetal cases are looking it up from DNS anyway and there's no way to override that AFAICT.

https://github.com/openshift-metal3/dev-scripts/blob/aa9e3a5dd2e0b5731b58d6f6112db7fda2ce3c95/06_create_cluster.sh#L24

Reply:
Got it, thanks.  I've updated to remove it.  I did leave the $API_VIP env variables, as they're used elsewhere.